src/shared/lib/isSampleProfiled.ts

```diff
@@ -0,0 +1,28 @@
+import * as _ from 'lodash';
```

Comment:
unless you made changes since you sent this to me, just scrap this file for the one that will conflict w it when you rebase